src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDaoImpl;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String SEPARATOR = ",";
+    private static final String HEADER = "fruit,quantity";
+
+    @Override
+    public String generateReport() {
+        StringBuilder stringBuilder = new StringBuilder(HEADER);
```

Comment:
do not use such naming(stringBuilder), how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder(HEADER);
```